lib/vagrant-libvirt/config.rb

```diff
@@ -529,6 +530,7 @@ def finalize!
         @graphics_ip = '127.0.0.1' if @graphics_ip == UNSET_VALUE
         @video_type = 'cirrus' if @video_type == UNSET_VALUE
         @video_vram = 9216 if @video_vram == UNSET_VALUE
+        @sound_type = 'ich6' if @sound_type == UNSET_VALUE
```

Comment:
I don't think we should default to adding a sound card. In most cases this will not be used.